src/Controllers/SearchController.cs

```diff
@@ -68,13 +74,42 @@ public SearchController(ILogger<SearchController> logger, RepoBrowserContext con
         }
     }
 
+    private Dictionary<string, Repository> FilterRepositories(string[]? repositoryNames, Dictionary<string, Repository> repositories)
+    {
+        if (repositoryNames != null && !repositoryNames.All(s => string.IsNullOrEmpty(s)))
```

Comment:
PP: Kann vereinfacht werden zu 
```suggestion
        if (repositoryNames != null && !repositoryNames.All(string.IsNullOrEmpty))
```